examples/decoding/plot_decoding_spoc_CMC.py

```diff
@@ -82,3 +82,10 @@
 plt.legend()
 mne.viz.tight_layout()
 plt.show()
+
+##############################################################################
+# Plot the contributions to the detected components (i.e., the forward model)
+
+spoc.fit_transform(X, y)
```

Comment:
why not just fit? You should not need to transform for this to work. Con you check?

Reply:
Yes, fit works just fine -- I'll update.